controllers/AuthController.js

```diff
@@ -153,6 +155,19 @@ export default class AuthController {
         id, isVerified, type
       } = createdUser[0];
 
+      // eslint-disable-next-line no-underscore-dangle
+      if (createdUser[0]._options.isNewRecord) {
+        const avatar = gravatar.url(email, {
```

Comment:
One of the details we get from authentication is a user's profile image. Is it possible to work with that?

Reply:
Yes, but if the user doesn't have a profile picture, gravatar could help us generate one for them

Reply:
Line 160 doesn't look like it's generated only when there is no avatar

Reply:
No, it just pulls the image from the user's email profile and if it doesn't exist it creates a generic one. That's how gravatar does it

Reply:
Ok, thanks for the clarification